storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/RocksDbInstanceFactory.java

```diff
@@ -147,6 +147,8 @@ private static DBOptions createDBOptions(
             .setMaxBackgroundJobs(configuration.getMaxBackgroundJobs())
             .setDbWriteBufferSize(configuration.getWriteBufferCapacity())
             .setMaxOpenFiles(configuration.getMaxOpenFiles())
+            .setBytesPerSync(1_048_576L) // 1MB
```

Comment:
both these settings should be removed as they were intentionally removed from the master and restored here by merge error

Reply:
Nice catch!